Library/Formula/telegraf.rb

```diff
@@ -0,0 +1,145 @@
+require "language/go"
+
+class Telegraf < Formula
+  desc "Server-level agent for InfluxDB"
+  homepage "https://influxdb.com"
+  url "https://github.com/influxdb/telegraf/archive/v0.1.1.tar.gz"
+  sha1 "86a4d00c9f8d34be3ac5b3a5a2db8eae5f3d8a14"
```

Comment:
Should be `sha256`. `brew audit --strict` will warn you about that, which is what gets ran against new submissions.